src/liballoc/collections/linked_list.rs

```diff
@@ -878,6 +878,58 @@ impl<T> LinkedList<T> {
         unsafe { self.split_off_after_node(split_node, at) }
     }
 
+    /// Removes the element at the given index and returns it.
+    ///
+    /// This operation should compute in O(n) time.
+    ///
+    /// # Panics
+    /// Panics if at >= len
```

Comment:
```suggestion
    /// Panics if the given index is >= len
```
Totally optional. When I first read this, I thought it was a typo until I got down to the signature.